db/migrate/20210911182500_add_columns_for_case_development_analytics.rb

```diff
@@ -0,0 +1,15 @@
+class AddColumnsForCaseDevelopmentAnalytics < ActiveRecord::Migration[6.1]
+  def change
+    remove_column :monitoree_snapshots, :isolation_to_exposure, :integer
+    remove_column :monitoree_snapshots, :exposure_to_isolation, :integer
+
+    add_column :monitoree_snapshots, :exposure_to_isolation_active, :integer
+    add_column :monitoree_snapshots, :exposure_to_isolation_not_active, :integer
+    add_column :monitoree_snapshots, :cases_closed_in_exposure, :integer
+    add_column :monitoree_snapshots, :isolation_to_exposure, :integer
```

Comment:
@tstrass I believe this was something you implemented, is there a reason?

Reply:
Yeah we just want to destroy the old data, since the definition has changed. Definitely worth adding a comment in here explaining the whole migration beyond that detail.

Reply:
added here: https://github.com/SaraAlert/SaraAlert/pull/1085/commits/5f4d6b583878d607a0f11887bcf0739f09f58946